Gaming The Syllabus/Assets/Scripts/Combat/Action Effects/DamageTarget.cs

```diff
@@ -7,7 +7,7 @@
 
 public class DamageTarget : ActionEffect
 {
-    private int damageAmount;
+    public int damageAmount;
```

Comment:
I'm mostly okay with this being public, but we should still probly just make it a property with private set